include/boost/math/distributions/poisson.hpp

```diff
@@ -554,7 +554,6 @@ namespace boost
 // for this distribution have been defined, in order to
 // keep compilers that support two-phase lookup happy.
 #include <boost/math/distributions/detail/derived_accessors.hpp>
-#include <boost/math/distributions/detail/inv_discrete_quantile.hpp>
```

Comment:
There is a genuine mistake here, based on the comment in the file, it looks like we should be removing the *first* rather than last occurrence of the #include.  My bad on this one!

Reply:
Ok so I removed that instead and now it doesn't build